site/_includes/partials/post-headline.njk

```diff
@@ -12,6 +12,8 @@
     <span>•</span>
     Updated on <time>{{ helpers.formatDateLong(updated, locale) }}</time>
   {% endif %}
+  <br>
+  {% LanguageList page.url, site, collections, locale %}
```

Comment:
This has a couple of problems. If the article has no date, then this won't be shown at all.

You can change this whole section a bit to look maybe like (noting we can drop `.dates` because it just sets a color, wich we can instead do with utility class "color-secondary-text" instead):

```js
{% if date %}
<p class="type--caption color-secondary-text">...</p>
{% endif %}
{% set languageListHtml %}{% LanguageList ... %}{% endset %}
{% if languageListHtml %}
<p class="type--caption color-secondary-text">...</p>
{% endif %}
```